gulpfile.js

```diff
@@ -0,0 +1,40 @@
+var gulp = require('gulp');
```

Comment:
Instead of gulp, consider using a zero-config approach. Tools like `parcel` can help you out ;) Should not take long to setup.

Reply:
Wasn't aware of this tool. Will check it. Thanks.